### PR TITLE
Consolidated messages

### PR DESCRIPTION
**Pull request by livlab**
Consolidated the phrasing for messages 2 through 8. I hope this works! 😬

### CODE REVIEW

**Comment by livlab** [comment]
I also wanted to add  (median _%s_) to the end of the message, but not sure how to get the a) make it clear that 'time' is median and create another variable to be 'average'.

File: util/slack.py [hunk 1]
```diff
@@ -118,53 +118,13 @@ def get_linger_time_message_and_attachment(self, story, stats_per_slug, time_buc
                 time
             )
 
-        if time_bucket == 'hour 8':
-            message = ("%s hours in and _%s_ had users spending about *%s* "
-                "interacting with the graphic.") % (
+        if time_bucket == '8 hours' or time_bucket == '12 hours' or time_bucket == 'A day' or time_bucket == 'A day and a half' or time_bucket == 'Two days' or time_bucket == 'Two days and a half':
```
Comment (hampelm) [inline util/slack.py]: If this covers all scenarios after the first message, you could reduce this to an "else" statement. Eg:

```
if time_bucket == 'hour 4':
    # Send "I just started tracking" message. 

else: 
    # The new generic message
```

File: util/time.py [hunk 2]
```diff
@@ -36,10 +36,10 @@ def time_bucket(t):
         seconds_since_second_day = TimeTools.seconds_since(second_day_midnight_after_publishing)
 
         if seconds_since_second_day > 15 * 60 * 60: # 15 hours
-            return 'day 2 hour 15'
+            return 'Two days and a half'
```
Comment (hampelm) [inline util/time.py]: I think this would read more smoothly as "Two and a half days"

**Comment by livlab** [comment]
if/else is better, thank you. I had forgotten we need a final statement on the last one so added that back. let me know what you think.

**Comment by hampelm** [comment]
Looks good. Could you change up "Two days and a half" to "Two and a half days", and then this is ready to go!

**Comment by livlab** [comment]
oh, of course! d'oh.

**Comment by hampelm** [comment]
👍

**Pull request merged**